### DISCUSSION TITLE
Custom Model

### DISCUSSION THREAD

**Discussion by SorrHash** [CarlGao4/Demucs-Gui#66] [Q&A] [answered]
Hi, I want to use this custom model was made by ZFTurbo (https://github.com/ZFTurbo/MVSEP-CDX23-Cinematic-Sound-Demixing)

So.. I would like to know, is possible to use custom model in this GUI ? or not ? if possible, so how?

thx in your advance

**Comment by CarlGao4** [comment, marked as answer]
https://github.com/CarlGao4/Demucs-Gui/blob/main/usage.md#load-custom-models

**Reply by SorrHash** [reply]
it works and opens, but when i started separation, it's failed
![dwedew](https://github.com/CarlGao4/Demucs-Gui/assets/114016987/024e3aab-69c9-44a5-abd4-01441696045f)
IDK why, any help?

**Reply by CarlGao4** [reply]
Please attach a copy of log file

**Reply by SorrHash** [reply]
[20240516_135212_demucs_gui_log.log](https://github.com/CarlGao4/Demucs-Gui/files/15334271/20240516_135212_demucs_gui_log.log)

**Reply by CarlGao4** [reply]
It seems that you've removed some part of your log file. Sorry but the information I can see in this file is not enough to know what happened.

**Reply by SorrHash** [reply]
i just drag it from here to sent it for you.
![dewdwe](https://github.com/CarlGao4/Demucs-Gui/assets/114016987/0b2b6ecd-1a30-4ec5-8e7e-4d6a4f66b03a)
i didn't remove any part from it.

**Reply by SorrHash** [reply]
Anyway, thx for you❤

**Reply by CarlGao4** [reply]
Then...
This is so strange
It seems that something unexpected happened when initializing log file so it contains less useful data

**Reply by CarlGao4** [reply]
Found issue. You can just change `segment` from 8.7 to 8.6 and everything will be OK. Because the largest available is 8.68, however I set increment of segment spinbox to 0.1, so the default segment changed to 8.7, which is larger than 8.68 and caused the error.

**Reply by SorrHash** [reply]
![eewrw](https://github.com/CarlGao4/Demucs-Gui/assets/114016987/a2762efd-ccb3-44a7-964a-e44e86645099)
A HUGE THANX!❤❤

**Reply by CarlGao4** [reply]
And you can use this Bag of models to achieve better results. To extract it, please remove the `.zip` suffix of downloaded files

[cinematic_sound.zip.001](https://github.com/CarlGao4/Demucs-Gui/files/15334687/cinematic_sound.zip.001.zip)
[cinematic_sound.zip.002](https://github.com/CarlGao4/Demucs-Gui/files/15334691/cinematic_sound.zip.002.zip)
[cinematic_sound.zip.003](https://github.com/CarlGao4/Demucs-Gui/files/15334706/cinematic_sound.zip.003.zip)
[cinematic_sound.zip.004](https://github.com/CarlGao4/Demucs-Gui/files/15334742/cinematic_sound.zip.004.zip)
[cinematic_sound.zip.005](https://github.com/CarlGao4/Demucs-Gui/files/15334733/cinematic_sound.zip.005.zip)
[cinematic_sound.zip.006](https://github.com/CarlGao4/Demucs-Gui/files/15334736/cinematic_sound.zip.006.zip)
